src/components/Campaign.tsx

```diff
@@ -53,13 +71,37 @@ export const Campaign = () => {
           <Header>
             <EditorTopBar
               data-testid={editorTopBarTestId}
-              onSave={save}
               title={
                 campaignContentQuery.data?.campaignName
                   ? campaignContentQuery.data.campaignName
                   : ""
               }
-            />
+            >
+              <ul className="ed-header-list">
+                <li>
+                  {campaignContentQuery.data?.type === "unlayer" ? (
+                    <button
+                      type="button"
+                      onClick={showModal}
+                      className="dp-button button-medium primary-green"
+                    >
+                      <FormattedMessage id="save_template" />
+                    </button>
+                  ) : (
+                    ""
+                  )}
+                </li>
+                <li>
+                  <button
+                    type="button"
+                    onClick={save}
+                    className="dp-button button-medium primary-green"
+                  >
+                    <FormattedMessage id="save" />
+                  </button>
+                </li>
+              </ul>
```

Comment:
I think that there are more elegant alternatives for this `<ul>`/`<li>` structure, for example:

```suggestion
              <EditorTopBarAction>
                  {campaignContentQuery.data?.type === "unlayer" ? (
                    <button
                      type="button"
                      onClick={showModal}
                      className="dp-button button-medium primary-green"
                    >
                      <FormattedMessage id="save_template" />
                    </button>
                  ) : (
                    ""
                  )}
              </EditorTopBarAction>
              <EditorTopBarAction>
                  <button
                    type="button"
                    onClick={save}
                    className="dp-button button-medium primary-green"
                  >
                    <FormattedMessage id="save" />
                  </button>
              </EditorTopBarAction>
```

Or

```suggestion
                  {campaignContentQuery.data?.type === "unlayer" ? (
                    <button
                      type="button"
                      onClick={showModal}
                      className="dp-button button-medium primary-green"
                    >
                      <FormattedMessage id="save_template" />
                    </button>
                  ) : (
                    ""
                  )}
                  <button
                    type="button"
                    onClick={save}
                    className="dp-button button-medium primary-green"
                  >
                    <FormattedMessage id="save" />
                  </button>
```

This last option is easier but more automagical because you have to change `<EditorTopBar>` in this way:

```diff
-        {children ? <li>{children}</li> : ""}
+        {children ? {children.map(child => (<li>child</li>))} : ""}
```

Reply:
I don't have problem with current structure.

Reply:
I do 😛

Reply:
Maybe it is better to talk about it, probably we are seeing different things.

Reply:
I like this approach. 💪🏻